<!---->
<launch>
	<node pkg="joy" type="joy_node" name="joy_node" output="screen">
			<param name="joy_node/dev" value="/dev/input/js0"/>
			<param name="deadzone" value="0.001"/>
			<param name="autorepeat_rate" value="5.0"/>
			<param name="coalesce_interval" value="0.001"/>
		
	</node>

	<node pkg="gamepad" type="joystick_core" name="joy" output="screen">
			<param name="joy_sub" value="/joy"/>
			<param name="cmd_vel" value="/cmd_vel_joy"/>
			<param name="cmd_vel_frame" value="/base_link"/>
	</node>

</launch>

